project/panel.rkt

```diff
@@ -150,13 +151,27 @@ modifier author: Lîm Tsú-thuàn(GitHub: @dannypsnl)
     (define (remove-path-and-refresh btn event)
       (delete-directory/files (send view get-cur-selected-file) #:must-exist? #f)
       (send view reset-directory (send current-project get)))
+    (define (rename-path-and-refresh btn event)
+      (define selected-dir (send view get-cur-selected-dir))
+      (define selected-file (send view get-cur-selected-file))
+      (define name (get-text-from-user "new name of selected path?" ""))
```

Comment:
new name for

Reply:
by the way, give it init-val and #f parent. ref to https://docs.racket-lang.org/gui/Windowing_Functions.html?q=get-text-from-user#%28def._%28%28lib._mred%2Fmain..rkt%29._get-text-from-user%29%29